crates/node/core/src/args/metrics.rs

```diff
@@ -0,0 +1,22 @@
+use clap::Parser;
+use reth_cli_util::parse_socket_address;
+use std::net::SocketAddr;
+
+/// Telemetry exporters configuration.
+///
+/// - Prometheus: serves metrics over http.
+/// - OTLP: pushes metrics to an `OpenTelemetry` collector via http.
+#[derive(Debug, Clone, Default, Parser)]
+pub struct MetricsArgs {
+    /// Enable Prometheus metrics.
+    ///
+    /// The metrics will be served at the given interface and port.
+    #[arg(long, alias = "metrics.prometheus", value_name = "PROMETHEUS", value_parser = parse_socket_address, help_heading = "Metrics")]
```

Comment:
would be nice to not break existing setups due to changing the argument name, can we preserve backward compatibility with `--metrics` argument?